KuduSync.NET/KuduSync.cs

```diff
@@ -162,10 +163,13 @@ public void Run()
                 // If the file doesn't exist in the source, only delete if:
                 // 1. We have no previous directory
                 // 2. We have a previous directory and the file exists there
+                // 3. File is not a kudu created AppOffline.htm
 
                 // Trim the start destinationFilePath
                 string previousPath = FileSystemHelpers.GetRelativePath(destinationPath, destFile.FullName);
-                if (!sourceFilesLookup.ContainsKey(destFile.Name) && DoesPathExistsInManifest(previousPath, targetSubFolder))
+
+                bool isFileKuduAppOffline = _saveAppOffline && destFile.FullName.Equals(Path.Combine(_to, Program.AppOfflineFileName));
```

Comment:
Best to use `StringComparison.OrdinalIgnoreCase` for such comparisons

Reply:
Might also want to take `Path.Combine(_to, Program.AppOfflineFileName)` out of the loop since it's an invariant